src/vs/workbench/contrib/mergeEditor/browser/mergeEditorInput.ts

```diff
@@ -75,7 +75,13 @@ export class MergeEditorInput extends AbstractTextResourceEditorInput implements
 	}
 
 	override get capabilities(): EditorInputCapabilities {
-		return super.capabilities | EditorInputCapabilities.MultipleEditors | EditorInputCapabilities.Untitled;
+		let capabilities = super.capabilities | EditorInputCapabilities.MultipleEditors;
+
+		if (this.useWorkingCopy) {
+			capabilities |= EditorInputCapabilities.Untitled;
```

Comment:
My understanding is we only want `Untitled` to prevent auto save when we use a temp model. But for workspace, this will enable the `File - Revert` command.

Reply:
Right, thank you!